doc/api/cli.md

```diff
@@ -178,6 +178,16 @@ added: v8.0.0
 Enable loading native modules compiled with the ABI-stable Node.js API (N-API)
 (experimental).
 
+## `--abort-on-uncaught-exception`
+<!-- YAML
+added: v0.10
+-->
+
+Abort (dump core) when an uncaught exception is thrown. On most systems this will
+generate a core file, though `ulimit -c unlimited` may be necessary if resource
```

Comment:
I added this from an earlier review comment. Many node users aren't familiar with ulimit and core dumps, and I'm pretty sure we've seen issues raised claiming the feature is broken because there is no core dump, this attempts to preempt that.